### PR TITLE
Task4 keys and insert

### PR DESCRIPTION
**Pull request by ArBond**

### CODE REVIEW

File: lab-SQL/task4_keys-and-INSERT.sql [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		province.ModifiedDate
+FROM Person.StateProvince province
+JOIN Person.CountryRegion region
+ON province.CountryRegionCode = region.CountryRegionCode
+WHERE province.Name = region.Name AND province.CountryRegionCode NOT LIKE '%[0-9]%';
+GO
+
+--6
+ALTER TABLE dbo.StateProvince DROP COLUMN IsOnlyStateProvinceFlag;
+ALTER TABLE dbo.StateProvince ADD Population INT;
```
Comment (Glareone) [inline lab-SQL/task4_keys-and-INSERT.sql]: ALTER TABLE dbo.StateProvince ADD Population INT **NULL**

File: lab-SQL/task4_keys-and-INSERT.sql [hunk 2]
```diff
@@ -0,0 +1,98 @@
+USE  AdventureWorks2017;
+GO
+
+--#4.1
+
+--1
```
Comment (Glareone) [inline lab-SQL/task4_keys-and-INSERT.sql]: also let's add comments on English what every query do.

File: lab-SQL/task4_keys-and-INSERT.sql (hunk 1 above)
Comment (ArBond) [reply lab-SQL/task4_keys-and-INSERT.sql]: done

File: lab-SQL/task4_keys-and-INSERT.sql (hunk 2 above)
Comment (ArBond) [reply lab-SQL/task4_keys-and-INSERT.sql]: done

**Pull request merged**